### PR TITLE
remove side bar/update qset naming convention

### PR DESCRIPTION
**Pull request by rckim77**
-renamed 'organizationpage.html.erb' to 'show_subsets.html.erb'
-updated/added comments in qset_controller
-added 'show_questions.html.erb' view
-old 'questions' is now named 'mixed' (shows both questions and side bar)
-'questions' now refers to the newly added questions only page (removed side bar)
-updated modal UI to reflect these changes

### CODE REVIEW

**Comment by rckim77** [comment]
updated heroku app for testing: https://evening-scrubland-7568.herokuapp.com/

File: app/controllers/qsets_controller.rb [hunk 1]
```diff
@@ -38,8 +38,12 @@ def show
         @filter_mine = true
         @questions = @questions.where(user_id: current_user)
       end
+      # display question-only view
+      if @qset.settings(:permissions).qset_type == 'questions'
```
Comment (elliottyates) [inline app/controllers/qsets_controller.rb:42]: I've just realized that `qset_type` isn't really a 'permission' per se, is it? It's more like a generic qset-level setting, than a permission. We may want to consider moving it into a new `.settings(:key)` at some point for clarity, but for now it works and is at least consistent with the other qset-level settings -- just mentioning it to document it somewhere.

File: app/views/qsets/show_questions.html.erb [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  <%= render "qset_context" %>
+  <div id="generate-question-button-filter">
+    <%= render "generate_question" %>
+  </div>
+  <div id="all-mine-filter-container">
+    <%= render "all_mine_other_filter" %>
+  </div>
+  <div class="row">
+    <div class="col-sm-12">
+      <div id ="generate-question-button-filter-xs">
```
Comment (elliottyates) [inline app/views/qsets/show_questions.html.erb]: unnecessary space around `=`

File: app/views/qsets/show_questions.html.erb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      <div id="all-mine-filter-container-xs">
+        <%= render "all_mine_other_filter" %>
+      </div>
+      <%= render "question_list" %>
+    </div>
+  </div>
+</div>
+
+<%= render "modal_question" %>
+<%= render "modal_social" %>
```
Comment (elliottyates) [inline app/views/qsets/show_questions.html.erb]: We're no longer including a link to the social modal, so I think we don't need to render this.

**Comment by elliottyates** [comment]
I noticed two things. One is [a bug that I think is unrelated](https://trello.com/c/bztgrrzl/520-bug-sometime-there-are-no-questions-message-on-the-qset-page-has-a-confusing-message), but exposed by this work (so it should not hold back this code). The second is that the option in the UI is a little confusing-looking; perhaps we can clean it up a bit e.g. by spacing out the radio buttons? Right now it's hard to tell at a glance which radio button applies to which option/label.

screen shot 2016-02-24 at 10 24 52 am

**Comment by elliottyates** [comment]
Very helpful PR comments and code comments, nice work.

**Comment by rckim77** [comment]
updated heroku: https://evening-scrubland-7568.herokuapp.com

File: app/views/qsets/_modal_edit_qset.html.erb [hunk 4]
```diff
@@ -31,9 +31,11 @@
             </div>
             <div class="form-group">
               <%= permissions.label :qset_type, 'Qset type:' %>
-              <%= permissions.label :qset_type_questions, 'Questions' %>
+              <%= permissions.label :qset_type_mixed, 'Mixed', :class => 'qset-type-padding' %>
+              <%= permissions.radio_button :qset_type, 'mixed' %>
+              <%= permissions.label :qset_type_questions, 'Questions', :class => 'qset-type-padding' %>
               <%= permissions.radio_button :qset_type, 'questions' %>
-              <%= permissions.label :qset_type_subsets, 'Subsets' %>
+              <%= permissions.label :qset_type_subsets, 'Subsets', :class => 'qset-type-padding' %>
```
Comment (elliottyates) [inline app/views/qsets/_modal_edit_qset.html.erb:38]: FYI there is a more succinct way to do this. e.g. https://github.com/penzance/askup/blob/master/app/views/qsets/_modal_edit_qset.html.erb#L42

**Comment by elliottyates** [comment]
:+1:

**Pull request merged**